src/Lykke.Service.BalanceMismatches/Settings/BalanceMismatchesSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿using JetBrains.Annotations;
+using System.Collections.Generic;
+
+namespace Lykke.Service.BalanceMismatches.Settings
+{
+    [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
+    public class BalanceMismatchesSettings
+    {
+        public DbSettings Db { get; set; }
+
+        public CqrsSettings Cqrs { get; set; }
+
+        public List<HotWallet> HotWallets { get; set; }
```

Comment:
For the BIL integrations, all HW addresses could be obtained from the the Lykke.Service.BlockchainSettings. This service is not released yet, but it will released soon. Consider to use it in the future.

Reply:
Sure - will replace settings with a call to this service when it's available